go.mod

```diff
@@ -17,10 +19,10 @@ require (
 	github.com/rs/zerolog v1.32.0
 	github.com/slack-go/slack v0.14.0
 	github.com/spf13/cobra v1.8.0
-	github.com/stretchr/testify v1.9.0
-	github.com/testcontainers/testcontainers-go v0.27.0
-	golang.org/x/sync v0.8.0
-	golang.org/x/sys v0.24.0
+	github.com/stretchr/testify v1.11.0
+	github.com/testcontainers/testcontainers-go v0.40.0
```

Comment:
we update from 0.27 to 0.40

upgrades with listed breaking changes
- .28 - https://github.com/testcontainers/testcontainers-go/releases/tag/v0.28.0
- .31 - https://github.com/testcontainers/testcontainers-go/releases/tag/v0.31.0
- .33 - https://github.com/testcontainers/testcontainers-go/releases/tag/v0.33.0
- .35 - https://github.com/testcontainers/testcontainers-go/releases/tag/v0.35.0
- .36 - https://github.com/testcontainers/testcontainers-go/releases/tag/v0.36.0
- .38 - https://github.com/testcontainers/testcontainers-go/releases/tag/v0.38.0
- .40 - https://github.com/testcontainers/testcontainers-go/releases/tag/v0.40.0

Reply:
0.36 https://github.com/testcontainers/testcontainers-go/releases/tag/v0.36.0 comes with a docker library update, which is already included in this PR

other changes don't seem to affect us